app/javascript/crayons/CTAs/CTA.jsx

```diff
@@ -1,22 +1,47 @@
 import { h } from 'preact';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
-import { Link } from '@crayons';
+import { defaultChildrenPropTypes } from '../../common-prop-types/default-children-prop-types';
+import { Icon } from '@crayons';
 
 export const CTA = (props) => {
-  const { variant = 'default', className, ...otherProps } = props;
+  const {
+    children,
+    href = '#',
+    variant,
+    icon,
+    className,
+    ...otherProps
+  } = props;
 
   const classes = classNames('c-cta', {
     [`c-cta--${variant}`]: variant,
+    'c-cta--icon-left': icon && children,
     [className]: className,
   });
 
-  return <Link block className={classes} {...otherProps} />;
+  return (
+    <a href={href} className={classes} {...otherProps}>
+      {icon && (
+        <Icon
+          src={icon}
+          aria-hidden="true"
+          focusable="false"
+          className={classNames('c-cta__icon')}
```

Comment:
Refactor (blocking): Only one CSS class is being used. The `classNames` helper is not necessary.

```suggestion
          className="c-cta__icon"
```